writingIdomatic.py

```diff
@@ -0,0 +1,70 @@
+# /pythonPractice/writingIdomatic.py
+# -*- coding: UTF-8 -*-
+
+from __future__ import print_function
+
+print("\n--Let's Practice!--\n")
+
+# Notes:    When using 'if' statements do the error checking FIRST, then have the code that supposed to run normally, 
+#           This lets a reader easily skip to what should happen.
+# Ex: None
+
+
+# Notes:    Chain arguments in the 'if' statements if you can
+# Ex: None
+
+
+# Notes:    Don’t repeat a '==' comparison in an 'if' statement, use a list or set when you can. ‘x’ in ‘some_list’ can return true or false
+# Ex:
+
+is_generic_name = False
+name = 'Thomas'
+if name == 'Thomas' or name == 'Amir' or name == 'Brett':
```

Comment:
Using constants is good practice if you're typing something more than once. For example, I see 'Thomas' 3 times. If you needed to change that value for whatever reason, using a constant you'd only need to change it in one place.

Reply:
Makes sense, updated